src/components/AccountSettings.tsx

```diff
@@ -331,7 +330,26 @@ export const AccountSettings = () => {
                       </span>
                     )}
                   </div>
-                  <ORCiDButton sx={credentialButtonSX} />
+                  {orcid ? (
+                    <Button
+                      variant="outlined"
+                      sx={credentialButtonSX}
```

Comment:
very nit-picky, and really just a matter of preference but  _credentialButtonSX_ is  a single prop definition {marginRight: '26px'} So I would consider moving it inline just for the sake of colocation. But then we'd have to repeat it twice for the ln336 &343 so I can see the argument for keeping it as is as well.
 May be an alternative would be to turn 
`<div className="credential-partition">` into `<Box className="credential-partition" sx={{'& button': {marginRight: '26px'}}}>` All in all I don't think it matters a whole lot one way or the other.